leetcodePython/SlidingWindow/slidingwindow_862.py

```diff
@@ -0,0 +1,29 @@
+from itertools import accumulate
+from typing import List
+from collections import deque
+
+
+class Solution:
+    def shortestSubarray(self, nums: List[int], k: int) -> int:
+        prefix_sum = [0] + list(accumulate(nums))
+        shortest = len(nums) + 1
+        increase_sum_index = deque()
+
+        for right in range(len(prefix_sum)):
+            right_sum = prefix_sum[right]
+
+            while increase_sum_index and right_sum <= prefix_sum[increase_sum_index[-1]]:
+                increase_sum_index.pop()
+
+            while increase_sum_index:
+                left = increase_sum_index[0]
+                if right_sum - prefix_sum[left] < k:
+                    break
+                shortest = min(shortest, right - left)
```

Comment:
Due to the setup of [prefix_sum](https://github.com/JangoBoogaloo/LeetCodeExcercise/pull/181/files#r1838628743). We are not `right - left + 1` anymore.